app/forms/new_phone_form.rb

```diff
@@ -90,6 +92,17 @@ def validate_not_duplicate
     errors.add(:phone, I18n.t('errors.messages.phone_duplicate'))
   end
 
+  BLOCKED_PHONE_TYPES = [
+    :premium_rate,
+    :shared_cost,
+  ].freeze
```

Comment:
Do we care toward a convention of class member order, i.e. constants at the top?

For example, https://rubystyle.guide/#consistent-classes

Reply:
Happy to move to the top if you feel strongly? My thinking was it's only used here and isn't exactly meant for use in other classes so it's kind of a private const